.github/workflows/github_CI.yml

```diff
@@ -0,0 +1,67 @@
+name: Mirror and run GitLab CI
+
+######################################################################
+# Runs CI on github that triggers CI to run on a mirror gitlab repo. #
+######################################################################
+
+on: # Controls when the action will run.
+  workflow_dispatch:
+  push: # Please add your branch down here if wanted. A build will be triggerd for each push.
+    branches: [HDLTopSim]
+  pull_request:
+    branches: [master, HDLTopSim]
+    
+env:
+  mirror_repo: "https://gitlab.cern.ch/cms-l1tk/firmware-hls" # gitlab mirror
+  current_branch: 'master' # fallback branch
+
+jobs:
+  trigger_gitlab_CI: # https://github.com/marketplace/actions/mirror-to-gitlab-and-run-gitlab-ci
+    name: trigger_gitlab_CI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: DebugPrint
+      run: |
+        echo "github.ref = ${{ github.ref }}"
+
+    - name: Redefine branch on pull_request
+      if: github.event_name == 'pull_request'
+      run: |
+        echo "current_branch=$(echo ${{ github.head_ref }})" >> $GITHUB_ENV
+
+    - name: Redefine branch on push
+      if: github.event_name == 'push'
+      run: |
+        echo "current_branch=$(echo ${{ github.ref }} | sed -E 's|refs/[a-zA-Z]+/||')" >> $GITHUB_ENV
+
+    - name: Trigger gitlab CI run
+      # Communicate between gitlab & github, and trigger gitlab CI run.
+      # (Forked from https://github.com/SvanBoxel/gitlab-mirror-and-ci-action).
+      #uses: aperloff/gitlab-mirror-and-ci-action@master
+      uses: cms-L1TK/gitlab-mirror-and-ci-action@master
+      #with:
```

Comment:
Same here

Reply:
I've removed the obsolete reference to  aperloff/gitlab-mirror-and-ci-action@master at L42. Is this what you are referring to?

Reply:
done